_spinnaker/policy_engine.md

```diff
@@ -49,6 +53,7 @@ The Policy Engine supports the following OPA server deployments:
 * An existing OPA cluster 
 * An OPA server deployed in an existing Kubernetes cluster with an Armory Spinnaker deployment. If you want to use this method, use the following YAML example to deploy the OPA server:
 
+This ConfigMap example creates an OPA deployment in the same namespace as your Spinnaker deployment: 
```

Comment:
This example is not for a `ConfigMap`, is for a `Deployment`. I'd suggest to change to
```
This example manifest creates an OPA deployment in the same namespace as your Spinnaker deployment: 
```